packages/core/src/components/badge/badge.tsx

```diff
@@ -15,11 +15,17 @@ export class AtomBadge {
     | 'danger'
     | 'neutral'
     | 'dark' = 'primary'
-
+  @Prop({ mutable: true }) size: 'default' | 'medium' | 'large' = 'default'
+  @Prop({ mutable: true }) shape?: 'round' | 'rectangle' = 'round'
```

Comment:
(question): With these properties, the badge component will be almost identical to the tag component.

If the developer wanted to use something larger, wouldn't it be better to use the tag? 🤔 

![image](https://github.com/user-attachments/assets/1ba594fc-5431-496f-a8d6-85180b4a6194)

Tag:

![image](https://github.com/user-attachments/assets/e1e60501-bb0c-4238-a5c1-3aad84490100)